db/seeds.d/109-katello-notification-blueprints.rb

```diff
@@ -22,7 +22,7 @@
   {
     group: N_('Subscriptions'),
     name: 'manifest_expired_warning',
-    message: N_('The manifest imported within Organization %{subject} is no longer valid.  Attempted CDN access returned Forbidden.  Refreshing the manifest may resolve this issue.'),
+    message: N_('The manifest imported within Organization %{subject} is no longer valid. Check the end date of any attached subscriptions and import a new manifest.'),
```

Comment:
I tried to make the message little more general since there are now two scenarios when this message sent